plugin.video.catchuptvandmore/resources/lib/channels/fr/bsmart.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
would be nice if at some point you can drop such large headers and adopt the spdx ones. Have a look into the kodi source code for an example

Reply:
I will look at that, thank you.